Makefile

```diff
@@ -12,18 +12,27 @@ IDENTIFIER=$(GOOS)-$(GOARCH)
 help:  ## Show available options with this Makefile
 	@grep -F -h "##" $(MAKEFILE_LIST) | grep -v grep | awk 'BEGIN { FS = ":.*?##" }; { printf "%-15s  %s\n", $$1,$$2 }'
 
+web:
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
+yarn:
+	@rm -rf web
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
 .PHONY: build
-build:
+build: 
 	GOOS=$(GOOS) GOARCH=$(GOARCH) $(GC) $(BUILD_NKND_PARAM) -o $(FLAGS)/nknd nknd.go
 	GOOS=$(GOOS) GOARCH=$(GOARCH) $(GC) $(BUILD_NKNC_PARAM) -o $(FLAGS)/nknc nknc.go
 
 .PHONY: crossbuild
-crossbuild:
+crossbuild: web
 	mkdir -p build/$(IDENTIFIER)
 	make build FLAGS="build/$(IDENTIFIER)"
+	@cp -a dashboard/web/dist build/$(IDENTIFIER)/web
 
 .PHONY: all
 all:  ## Build binaries for all available architectures
+	@rm -rf web
```

Comment:
We do we rm web here? If we just want a fresh build, I think we can use `make yarn` right?

Reply:
yarn is a PHONY target.
We don't expect to `yarn build` repeatedly for each crossPlatform during `make all`

Reply:
On the other hand, `yarn build` just only overwrite the output files with same path. It would not delete those files which was obsolete.